rpc/src/lib.rs

```diff
@@ -452,9 +452,9 @@ pub mod v1 {
     /// V1 Replica autogenerated grpc code.
     pub mod replica {
         pub use super::pb::{
-            replica_rpc_client, CreateReplicaRequest, DestroyReplicaRequest, ListReplicaOptions,
-            ListReplicasResponse, Replica, ReplicaSpaceUsage, ShareReplicaRequest,
-            UnshareReplicaRequest,
+            destroy_replica_request, replica_rpc_client, CreateReplicaRequest,
```

Comment:
Why are there 2 types of destroy replica request?

Reply:
the new one is not a request type, it's a module containing the `oneof` type which is declared within the destroy replica message.